src/status_im/ipfs/core.cljs

```diff
@@ -31,18 +31,18 @@
 
 (fx/defn add
   "Add `value` on ipfs, and returns its b58 encoded CID"
```

Comment:
This changes are not needed probably, we can keep them if useful, but they might have been left over of the ipfs implementation.